tests/unit/test-max-input-nodes.cpp

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (C) 2011-2013 Karlsruhe Institute of Technology
+ *
+ * This file is part of Ufo.
+ *
+ * This library is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation, either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "config.h"
```

Comment:
All that was needed was to swap these includes, gobject handles it gracefully. The rest of ufo also depends on it, so it's fine.

Reply:
But in CMake -> GnuMakefile it worked with the other order of the includes :confused: . 
I think we have to write tests for our build-systems as well :wink: .

Reply:
Becuase you created your "guard", which was inconsistent with meson so I dropped it and got rid of the warning for both via the swap.